app/vmctl/influx.go

```diff
@@ -27,6 +28,7 @@ func newInfluxProcessor(ic *influx.Client, im *vm.Importer, cc int, separator st
 		im:        im,
 		cc:        cc,
 		separator: separator,
+		quite:     quite,
```

Comment:
`quite` looks like typo, may `quit` ?

 I think, that mustStop or shouldStop, sounds better.

Reply:
fixed typo